packages/teams-js/test/internal/utils.spec.ts

```diff
@@ -276,4 +277,21 @@ describe('utils', () => {
       }
     });
   });
+
+  describe('validateAppIdIsGuid', () => {
```

Comment:
I don't think this matches exactly the set of unit tests we added in the host SDK. Is that intentional?

Reply:
There was a custom error test that doesn't apply here, but I added the "valid app ID, invalid v4 uuid" test